astro.config.mjs

```diff
@@ -62,172 +55,194 @@ export default defineConfig({
+                  collapsed: false,
+                  items: [
+                    {
+                      label: 'Monetization interfaces',
+                      link: '/developers/interfaces',
+                    },
+                    {
+                      label: 'Monetization events',
+                      link: '/developers/events',
+                    },
+                  ],
                 },
                 {
-                  label: 'Permissions-Policy: monetization',
-                  link: '/docs/references/permissions-policy-monetization',
+                  label: 'HTTP Headers',
+                  collapsed: true,
+                  items: [
+                    {
+                      label: 'Content Security Policy (CSP)',
+                      link: '/developers/csp',
+                    },
+                    {
+                      label: 'Permissions Policy',
+                      link: '/developers/permissions-policy',
+                    },
+                  ],
                 },
               ],
             },
             {
-              label: 'Deprecations',
-              link: '/docs/references/deprecations',
-            }
-          ],
-        },
-        {
-          label: 'Guides',
-          collapsed: true,
-          items: [
-            {
-              label: 'Add Web Monetization to a page',
-              link: '/docs/guides/monetize-page',
-            },
-            {
-              label: 'Remove ads',
-              link: '/docs/guides/remove-ads',
-            },
-            {
-              label: 'Provide exclusive content',
-              link: '/docs/guides/provide-exclusive-content',
+              label: 'Libraries',
+              link: '/developers/libraries',
             },
             {
-              label: 'Set up probabilistic revenue sharing',
-              link: '/docs/guides/set-up-probabilistic-revenue-sharing',
+              label: 'Publisher tools',
+              link: '/developers/tools',
             },
             {
-              label: 'Add a streaming payments counter',
-              link: '/docs/guides/add-a-streaming-payments-counter',
+              label: 'Web Monetization Specification',
+              link: 'https://webmonetization.org/specification/',
+              attrs: { target: '_blank' },
             },
           ],
         },
+        {
+          label: 'Compatible digital wallets',
+          link: '/wallets',
+        },
         {
           label: 'Resources',
           collapsed: true,
           items: [
             {
               label: 'Glossary',
-              link: '/docs/resources/glossary',
+              link: '/resources/glossary',
             },
             {
-              label: 'Libraries',
-              link: '/docs/resources/libraries',
+              label: 'Web Monetization extensions',
+              link: 'https://webmonetization.org/install/',
+              attrs: { target: '_blank' },
```

Comment:
Why does this page open in new tab?

Reply:
To keep the doc site open. I need to figure out if there's a way to show an icon to indicate it links out.

Reply:
There is a way, but I'll need Sarah to help apply it to all our doc sites. I've created an issue for her.

Reply:
Figured it out